cross-dom-bridge-erc20/index.js

```diff
@@ -47,68 +72,33 @@ const setup = async() => {
       l1SignerOrProvider: l1Signer,
       l2SignerOrProvider: l2Signer
```

Comment:
Also might want to add that these addresses will need eth on their respective chains no?

Reply:
I wrote that in the README.md,

>    1. Set `MNEMONIC` to point to an account that has ETH on the Goerli test network and the Optimism Goerli test network.

But I'll add it to the code too.